test/acceptance/features/steps/generic_testapp.py

```diff
@@ -74,3 +76,33 @@ def check_file_value(context, file_path):
 @step(u'File "{file_path}" is unavailable in application pod')
 def check_file_unavailable(context, file_path):
     context.application.assert_file_not_exist(file_path)
+
+
+@given(u'Test applications "{first_app_name}" and "{second_app_name}" is running')
+def are_two_apps_running(context, first_app_name, second_app_name, bindingRoot=None):
+    application1 = GenericTestApp(first_app_name, context.namespace.name)
+    if not application1.is_running():
+        print("application1 is not running, trying to import it")
+        application1.install(bindingRoot=bindingRoot)
+    context.application1 = application1
+
+    application2 = GenericTestApp(second_app_name, context.namespace.name)
+    if not application2.is_running():
+        print("application2 is not running, trying to import it")
+        application2.install(bindingRoot=bindingRoot)
+    context.application2 = application2
+
+
+@given(u'The common label "{label}" is set for both apps')
+def set_common_label(context, label):
+    context.application1.set_label(f"{label}")
+    context.application2.set_label(f"{label}")
+
+
+@then(u'The application env var "{name}" has value "{value}" in both apps')
+def check_env_var_value_in_both_apps(context, name, value):
+    found = polling2.poll(lambda: context.application1.get_env_var_value(name) == value, step=5, timeout=400)
+    assert found, f'Env var "{name}" should contain value "{value}" in {context.application1}'
```

Comment:
if we reach this line, then `found` is `true` always - otherwise Timeout exception occurs. Hence, this and the similar line at line 108 can be removed.

Reply:
Done. 5e91f6d96db0011740c9422b96e2de2d0a654551